pytest_enabler/__init__.py

```diff
@@ -24,7 +25,12 @@ def none_as_empty(ob):
 def read_plugins(filename):
     with open(filename) as strm:
         defn = toml.load(strm)
-    return defn["pytest"]["enabler"]
+    if "pytest" in defn and "enabler" in defn["pytest"]:
+        msg = "pytest-enabler configuration should use the `[tool.pytest-enabler]` "
+        msg += "table in pyproject.toml (`[pytest.enabler]` is now deprecated)."
+        warnings.warn(msg, DeprecationWarning)
+        return defn["pytest"]["enabler"]
+    return defn["tool"]["pytest-enabler"]
```

Comment:
I'd like to see this logic broken out as a compatibility shim that will be removed. I'll do that.

Reply:
Done in 746a84d.